### DISCUSSION TITLE
Facing error while running "npm install --legacy-peer-deps"

### DISCUSSION THREAD

**Discussion by Sudarshan-21** [processing/p5.js-web-editor#2960] [Q&A]
while running "npm install --legacy-peer-deps" I am getting the following error, can somebody help me with this
![airbnb err1](https://github.com/processing/p5.js-web-editor/assets/137636943/b77f8c53-a522-416b-98b6-c3b75ceeb7c7)
![airbnb err2](https://github.com/processing/p5.js-web-editor/assets/137636943/35850721-9abb-4bb0-91dc-fa5256f87b85)
![airbnb err3](https://github.com/processing/p5.js-web-editor/assets/137636943/29994dcf-ddd5-443d-af84-5383040dd320)

**Comment by Keshav-0907** [comment]
Have you tried deleting the 'node_modules' folder and the 'package-lock.json' file, and then reinstalling the node modules?

**Comment by ayeshag7** [comment]
Have you made sure that your node version is correct, i.e., `v16.14.2`? Maybe, [this](https://github.com/processing/p5.js-web-editor/discussions/2850) discussion will help you.
